backend/app/routes/admin/index.js

```diff
@@ -2,17 +2,28 @@ const router = require('express').Router({ mergeParams: true });
 const validationMiddleware = require('../../../helpers/middlewares/validation');
 const { authMiddleware } = require('../../../helpers/middlewares/auth');
 
-const { postSuperAdminSchema, getAdminsSchema, passwordChangeSchema } = require('./@validationSchema');
+const {
+  postSuperAdminSchema,
+  getAdminsSchema,
+  passwordChangeSchema,
+  forgotPasswordSchema,
+  resetPasswordSchema,
+} = require('./@validationSchema');
 const createSuperAdmin = require('./createSuperAdmin');
 const postSuperAdmin = require('./postSuperAdmin');
 const postAdmin = require('./postAdmin');
 const getAdmins = require('./getAdmins');
 const changePassword = require('./changePassword');
+const forgotPassword = require('./forgotPassword');
+const resetPassword = require('./resetPassword');
 
 router.get('/', validationMiddleware(getAdminsSchema, 'query'), authMiddleware, getAdmins);
 router.get('/createSuperAdmin', createSuperAdmin);
+
 router.post('/superAdmin', validationMiddleware(postSuperAdminSchema), postSuperAdmin);
 router.post('/', validationMiddleware(postSuperAdminSchema), authMiddleware, postAdmin);
+router.post('/forgotpassword', validationMiddleware(forgotPasswordSchema), forgotPassword);
+router.post('/forgotpassword/:token', validationMiddleware(resetPasswordSchema), resetPassword);
```

Comment:
I think so it should be `/resetpassword/:token` contextually, since forgot password is the api which one hits to reset their password. Post that they will get an email with a link to reset the password. lmk if you are aligned with this

Reply:
Ya sure makes sense